spec/multiply_els_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require_relative '../lib/multiply_els'
+
+RSpec.describe 'multiply_els' do
```

Comment:
Now that we have these tests, it seems redundant to make the student test their inject method by doing this exercise. What do you think about including this as another test case in the my_inject_spec.rb file instead?

Reply:
Makes sense, sounds like a good idea to me!